[2023-04-14 12:25:25,516] INFO [Worker clientId=connect-1, groupId=connect-cluster] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:25,521] INFO [Worker clientId=connect-1, groupId=connect-cluster] Cancelled in-flight METADATA request with correlation id 2163 due to node 0 being disconnected (elapsed time since creation: 0ms, elapsed time since send: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:341)
[2023-04-14 12:25:25,521] INFO [Worker clientId=connect-1, groupId=connect-cluster] Group coordinator localhost:9092 (id: 2147483647 rack: null) is unavailable or invalid due to cause: session timed out without receiving a heartbeat response. isDisconnected: false. Rediscovery will be attempted. (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:960)
[2023-04-14 12:25:25,517] INFO [Producer clientId=producer-1] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:26,538] INFO [Worker clientId=connect-1, groupId=connect-cluster] Requesting disconnect from last known coordinator localhost:9092 (id: 2147483647 rack: null) (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:973)
[2023-04-14 12:25:25,826] INFO [Consumer clientId=consumer-connect-cluster-1, groupId=connect-cluster] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:26,550] INFO [Consumer clientId=consumer-connect-cluster-1, groupId=connect-cluster] Cancelled in-flight FETCH request with correlation id 12578 due to node 0 being disconnected (elapsed time since creation: 1770105ms, elapsed time since send: 1770105ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:341)
[2023-04-14 12:25:25,794] INFO [Producer clientId=producer-3] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:25,611] INFO [Producer clientId=producer-2] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:25,610] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Node 2147483647 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:26,555] INFO [Consumer clientId=consumer-connect-cluster-1, groupId=connect-cluster] Error sending fetch request (sessionId=1396257758, epoch=12551) to node 0: (org.apache.kafka.clients.FetchSessionHandler:602)
org.apache.kafka.common.errors.DisconnectException
[2023-04-14 12:25:25,562] INFO [AdminClient clientId=adminclient-10] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:26,555] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Group coordinator localhost:9092 (id: 2147483647 rack: null) is unavailable or invalid due to cause: session timed out without receiving a heartbeat response. isDisconnected: false. Rediscovery will be attempted. (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:960)
[2023-04-14 12:25:26,550] INFO [Worker clientId=connect-1, groupId=connect-cluster] Client requested disconnect from node 2147483647 (org.apache.kafka.clients.NetworkClient:324)
[2023-04-14 12:25:26,561] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Requesting disconnect from last known coordinator localhost:9092 (id: 2147483647 rack: null) (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:973)
[2023-04-14 12:25:26,563] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:25:26,564] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Cancelled in-flight METADATA request with correlation id 10770 due to node 0 being disconnected (elapsed time since creation: 953ms, elapsed time since send: 953ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:341)
[2023-04-14 12:25:26,569] INFO [Worker clientId=connect-1, groupId=connect-cluster] Discovered group coordinator localhost:9092 (id: 2147483647 rack: null) (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:897)
[2023-04-14 12:25:26,627] INFO [Worker clientId=connect-1, groupId=connect-cluster] Attempt to heartbeat with Generation{generationId=3, memberId='connect-1-484ca515-2ef0-423f-8b08-68fb5ab9fef1', protocol='sessioned'} and group instance id Optional.empty failed due to UNKNOWN_MEMBER_ID, resetting generation (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1217)
[2023-04-14 12:25:26,664] INFO [Worker clientId=connect-1, groupId=connect-cluster] Resetting generation and member id due to: encountered UNKNOWN_MEMBER_ID from HEARTBEAT response (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1016)
[2023-04-14 12:25:26,679] INFO [Worker clientId=connect-1, groupId=connect-cluster] Rebalance started (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:229)
[2023-04-14 12:25:26,685] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Discovered group coordinator localhost:9092 (id: 2147483647 rack: null) (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:897)
[2023-04-14 12:25:26,687] INFO [Worker clientId=connect-1, groupId=connect-cluster] (Re-)joining group (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:566)
[2023-04-14 12:25:26,693] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Attempt to heartbeat with Generation{generationId=3, memberId='connector-consumer-my-sink-order-connect-0-e72bf91f-b69d-41e2-aef5-524d215a7e18', protocol='range'} and group instance id Optional.empty failed due to UNKNOWN_MEMBER_ID, resetting generation (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:1217)
[2023-04-14 12:25:26,699] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Resetting generation and member id due to: encountered UNKNOWN_MEMBER_ID from HEARTBEAT response (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:1016)
[2023-04-14 12:25:26,701] INFO [Worker clientId=connect-1, groupId=connect-cluster] Request joining group due to: rebalance failed due to 'The group member needs to have a valid member id before actually entering a consumer group.' (MemberIdRequiredException) (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1063)
[2023-04-14 12:25:26,703] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Request joining group due to: encountered UNKNOWN_MEMBER_ID from HEARTBEAT response (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:1063)
[2023-04-14 12:25:26,715] INFO [Worker clientId=connect-1, groupId=connect-cluster] (Re-)joining group (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:566)
[2023-04-14 12:25:26,727] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Giving away all assigned partitions as lost since generation/memberID has been reset,indicating that consumer is in old state or no longer part of the group (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:819)
[2023-04-14 12:25:26,729] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Lost previously assigned partitions orders-0 (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:354)
[2023-04-14 12:25:26,738] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] (Re-)joining group (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:566)
[2023-04-14 12:25:26,738] INFO [Worker clientId=connect-1, groupId=connect-cluster] Successfully joined group with generation Generation{generationId=5, memberId='connect-1-31fe0675-aad9-4375-9512-44aafb70ed94', protocol='sessioned'} (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:627)
[2023-04-14 12:25:26,746] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Request joining group due to: need to re-join with the given member-id: connector-consumer-my-sink-order-connect-0-90024578-6467-481a-8f6e-0e25375d439c (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:1063)
[2023-04-14 12:25:26,749] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Request joining group due to: rebalance failed due to 'The group member needs to have a valid member id before actually entering a consumer group.' (MemberIdRequiredException) (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:1063)
[2023-04-14 12:25:26,750] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] (Re-)joining group (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:566)
[2023-04-14 12:25:26,753] INFO [Worker clientId=connect-1, groupId=connect-cluster] Successfully synced group in generation Generation{generationId=5, memberId='connect-1-31fe0675-aad9-4375-9512-44aafb70ed94', protocol='sessioned'} (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:802)
[2023-04-14 12:25:26,753] INFO [Worker clientId=connect-1, groupId=connect-cluster] Joined group at generation 5 with protocol version 2 and got assignment: Assignment{error=0, leader='connect-1-31fe0675-aad9-4375-9512-44aafb70ed94', leaderUrl='http://10.111.150.68:8083/', offset=6, connectorIds=[my-sink-order-connect], taskIds=[my-sink-order-connect-0], revokedConnectorIds=[], revokedTaskIds=[], delay=0} with rebalance delay: 0 (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2256)
[2023-04-14 12:25:26,754] WARN [Worker clientId=connect-1, groupId=connect-cluster] Catching up to assignment's config offset. (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1519)
[2023-04-14 12:25:26,755] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Successfully joined group with generation Generation{generationId=5, memberId='connector-consumer-my-sink-order-connect-0-90024578-6467-481a-8f6e-0e25375d439c', protocol='range'} (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:627)
[2023-04-14 12:25:26,768] INFO [Worker clientId=connect-1, groupId=connect-cluster] Current config state offset 4 is behind group assignment 6, reading to end of config log (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1584)
[2023-04-14 12:25:26,769] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Finished assignment for group at generation 5: {connector-consumer-my-sink-order-connect-0-90024578-6467-481a-8f6e-0e25375d439c=Assignment(partitions=[orders-0])} (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:705)
[2023-04-14 12:25:26,779] INFO [Worker clientId=connect-1, groupId=connect-cluster] Finished reading to end of log and updated config snapshot, new config log offset: 6 (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1609)
[2023-04-14 12:25:26,782] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Successfully synced group in generation Generation{generationId=5, memberId='connector-consumer-my-sink-order-connect-0-90024578-6467-481a-8f6e-0e25375d439c', protocol='range'} (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:802)
[2023-04-14 12:25:26,790] INFO [Worker clientId=connect-1, groupId=connect-cluster] Starting connectors and tasks using config offset 6 (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1659)
[2023-04-14 12:25:26,800] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Notifying assignor about the new Assignment(partitions=[orders-0]) (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:300)
[2023-04-14 12:25:26,800] INFO [Worker clientId=connect-1, groupId=connect-cluster] Finished starting connectors and tasks (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1687)
[2023-04-14 12:25:26,800] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Adding newly assigned partitions: orders-0 (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:312)
[2023-04-14 12:25:26,803] INFO [my-sink-order-connect|task-0] [Consumer clientId=connector-consumer-my-sink-order-connect-0, groupId=connect-my-sink-order-connect] Setting offset for partition orders-0 to the committed offset FetchPosition{offset=99, offsetEpoch=Optional.empty, currentLeader=LeaderAndEpoch{leader=Optional[localhost:9092 (id: 0 rack: null)], epoch=0}} (org.apache.kafka.clients.consumer.internals.ConsumerCoordinator:968)
[2023-04-14 12:35:26,570] INFO [AdminClient clientId=adminclient-10] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:40:26,689] INFO [AdminClient clientId=adminclient-10] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:45:26,815] INFO [AdminClient clientId=adminclient-10] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:50:26,933] INFO [AdminClient clientId=adminclient-10] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:937)
[2023-04-14 12:52:15,990] INFO [Producer clientId=producer-3] Resetting the last seen epoch of partition connect-configs-0 to 0 since the associated topicId changed from null to T_zl79gQQXiH6jHAePBYSw (org.apache.kafka.clients.Metadata:402)
[2023-04-14 12:52:15,994] INFO [Worker clientId=connect-1, groupId=connect-cluster] Session key updated (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2124)
